packages/backend/src/config/app.ts

```diff
@@ -74,7 +74,7 @@ export const Config = {
   ),
   authServerIntrospectionUrl: envString(
     'AUTH_SERVER_INTROSPECTION_URL',
-    'http://127.0.0.1:3007'
+    'http://127.0.0.1:3007/introspect'
```

Comment:
Yes, because `/auth` is currently hardcoded on the [spec's server url.](https://github.com/interledger/open-payments/blob/master/auth-server-open-api-spec.yaml#L13), which the OpenAPI validation doesn't take into account (only sees the `/introspect` path).
I was thinking https://github.com/interledger/open-payments/issues/123 be an optional for changing the spec endpoint to `/introspect`, which seems to suggested in the [GNAP spec](https://datatracker.ietf.org/doc/html/draft-ietf-gnap-resource-servers#section-3.3).

Reply:
I see. I commented on the linked issue.